### DISCUSSION TITLE
Is TrackIR supported?

### DISCUSSION THREAD

**Discussion by Dunravin** [opentrack/opentrack#1865] [Q&A]
Hello, I put the hatchet in Windows 11, it's buried in the back yard...

I'm on Arch now and looking for a solution to make my TrackIR 5 work. I'm here after much frustration with linuxtrack because there only some Aur packages and they are not maintained and neither are working.

Can anyone tell me if TrackIR hardware will work with OpenTrack?

Many thanks :D

**Comment by NlGHTLORD** [comment]
It will not.
